### ISSUE TITLE
Release 1.0.0

### ISSUE THREAD

**Issue by AlanFoster** [onefinestay/react-daterange-picker#92]
Once 0.12.2 is released (#91) we should merge #79 and support React 0.14.x.
I feel that as part of this release we should update our version to 1.0.0, following semver semantics.

/cc @AndrewIngram @jkimbo Could you please review, merge and release as appropriate, as I do not have the permissions to do so.

**Comment by thathenderson** [comment]
Any chance this will be released soon?

**Comment by nicoecheza** [comment]
+1 to @thathenderson

**Comment by thathenderson** [comment]
Saw everything was merged to master! Awesome! Could we get the package file updated too?

**Comment by goldensunliu** [comment]
until this is closed. install it straight from master:
``` json
"react-daterange-picker": "git+https://github.com/onefinestay/react-daterange-picker.git"
```

**Comment by AlanFoster** [comment]
This should be resolved now, thanks to the [v1.0.0 release](https://github.com/onefinestay/react-daterange-picker/releases/tag/v1.0.0)
